tests/blockchain/Test_BlockchainAPI.cpp

```diff
@@ -548,8 +548,6 @@ TEST_F(Test_BlockchainAPI, invalid_nym)
     EXPECT_EQ(list.size(), 0);
     EXPECT_EQ(list.count(accountID), 0);
 
-    loaded = false;
```

Comment:
EXPECT_FALSE does not stop the test, so assuming loaded to be false here is wrong. Please explain or revert the change.

Reply:
good catch i will create another boolean for this